### PR TITLE
Create swapfile artifact

### PR DESCRIPTION
**Pull request by ryanvog**
Verified with Ubuntu, CentOS

### CODE REVIEW

File: Artifacts/linux-swapfile/Artifactfile.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2015-01-01/dtlArtifacts.json",
+    "title": "Swap file",
+    "description": "Configure swap file on Linux",
+    "tags": [
+		"Linux"
```
Comment (drwill-ms) [inline Artifacts/linux-swapfile/Artifactfile.json]: do you care about spaces/tabs?

File: Artifacts/linux-swapfile/Artifactfile.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    ],
+    "iconUri": "",
+    "targetOsType": "Linux",
+    "parameters": {
+        "size": {
+            "type": "string",
+            "displayName": "Size",
+            "description": "The size of the swap file in bytes.  Suffixes k, m, g, t, p, or e correspond to KiB, MiB, GiB, etc. Exmaple: 5g = 5GiB, 100m = 100MiB",
```
Comment (drwill-ms) [inline Artifacts/linux-swapfile/Artifactfile.json]: misspelled example

Comment (drwill-ms) [inline Artifacts/linux-swapfile/Artifactfile.json]: What does the "i" in "MiB" and other acronyms stand for?

File: Artifacts/linux-swapfile/Artifactfile.json (hunk 1 above)
Comment (ryanvog) [inline Artifacts/linux-swapfile/Artifactfile.json]: Fixed.

File: Artifacts/linux-swapfile/Artifactfile.json (hunk 2 above)
Comment (ryanvog) [inline Artifacts/linux-swapfile/Artifactfile.json]: Fixed

**Comment by drwill-ms** [comment]
looks great

**Pull request merged**